src/Stratis.Bitcoin/P2P/PeerDiscoveryLoop.cs

```diff
@@ -94,8 +100,18 @@ public void DiscoverPeers(IConnectionManager connectionManager)
             if (!connectionManager.Parameters.PeerAddressManagerBehaviour().Mode.HasFlag(PeerAddressManagerBehaviourMode.Discover))
                 return;
 
+            this.seedAndDnsAttempted = DateTime.Now;
```

Comment:
You don't need this, just set the interval on the loop to 12 hours ;)

Reply:
As discussed, we'll do it hourly as a compromise for the extreme case that the DNS seeds aren't contactable in the first loop execution.